3-enrich/scala-common-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/adapters/registry/IgluAdapter.scala

```diff
@@ -47,11 +48,73 @@ object IgluAdapter extends Adapter {
 
   // Tracker version for an Iglu-compatible webhook
   private val TrackerVersion = "com.snowplowanalytics.iglu-v1"
+  private val VendorName = "Iglu"
 
   // Create a simple formatter function
   private val IgluFormatter: FormatterFunc = buildFormatter() // For defaults
 
   /**
+   * Attempts to parse a json string into a JValue
+   * example: {"p":"app"} becomes JObject(List((p,JString(app))))
+   *
+   * @param jsonStr The string we want to parse into a JValue
+   * @return a Validated JValue or a NonEmptyList Failure
+   *         containing a JsonParseException
+   */
+  private[registry] def parseJson(jsonStr: String): Validated[JValue] =
+    try {
```

Comment:
Can we move this function to `Adapter.scala` to cut down on code duplication.  Potentially also rename to `parseJsonSafe`.  This function can then be used in the `PingdomAdapter` as well or anywhere else we need to safely parse JSON